manager/wasabi_clients/wasabi_client_v2_old.py

```diff
@@ -0,0 +1,41 @@
+from .wasabi_client_base import WasabiClientBase, WALLET_NAME
+from time import sleep, time
+from .client_versions_enum import VersionsEnum
+
+
+class WasabiClientV2Old(WasabiClientBase):
```

Comment:
I suggest using a different naming that Old/New. It may seem that old is something that we do not use anymore, yet that is not the case (it just uses an older API) and it also complicates naming for newer versions if they will make another API change. Perhaps we could use something like `WasabiClientV2` for 2.0.0-2.0.3 and `WasabiClientV204` for 2.0.4+?

Reply:
renamed as proposed. I will later add description to clarify for which versions of wallet wasabi each class is used